toolkit/scripts/pkggen.mk

```diff
@@ -223,6 +223,7 @@ $(STATUS_FLAGS_DIR)/build-rpms.flag: $(preprocessed_file) $(chroot_worker) $(go-
 		$(if $(filter-out y,$(CLEANUP_PACKAGE_BUILDS)),--no-cleanup) \
 		$(if $(filter y,$(DELTA_BUILD)),--delta-build) \
 		$(if $(filter y,$(USE_CCACHE)),--use-ccache) \
+		$(if $(filter y,$(ALLOW_TOOLCHAIN_REBUILDS)),--allow-prebuilt-rebuilds) \
```

Comment:
Please re-name `allow_prebuilt-rebuilds` to align with the `make` argument. I'm also fine with giving a different name but it should still make it clear, that this is about not treating toolchain packages rebuilds as erroneous. Rebuilds or prebuilt packages were always OK, so this is currently confusing.